recipes/tmuxp/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "tmuxp" %}
+{% set version = "1.45.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 23b3ff0a8862a44c2bc687a553f78f4afd8f1e03371d775555a7699ed54572b4
+
+build:
+  # Seeing weird errors like the following on Windows, along with weird encoding.
+  # https://learn.microsoft.com/en-us/cpp/error-messages/tool-errors/c-runtime-error-r6033?view=msvc-170
+  script:
+    - {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python
```

Comment:
```suggestion
    - python >=3.8
```

Reply:
We need run and host with the same python constraint.

Reply:
Sorry I forgot to push that one.